docs/en/03_How_tos/preparing_your_site_for_active_dr.md

```diff
@@ -6,12 +6,14 @@ summary: How to Prepare your website to use Active DR infrastructure.
 This guide describes the steps needed to get your site ready for Active Disaster Recovery stacks. None of these
 modifications are needed on regular stacks.
 
-Active DR stacks have two properties:
+Active DR stacks have the following properties:
 
  * They are load-balanced: the traffic is served from two datacentres, increasing the maximum potential capacity of the
  stack.
- * They are highly-available: if one datacentre / node exhibits a problem and is not reachable, or emits 5xx HTTP status
- code, this node will be pulled out of the pool and all traffic will be redirected to the other datacentre.
+ * They are highly-available. If one data centre / node exhibits a problem and is not reachable, or emits 5xx HTTP status
+ codes, this node will be pulled out of the pool and all traffic will be redirected to the other data centre.
+ * The databases on each node is synchronised seamlessly by the infrastructure.
+ * Files are synchronised from the ./www/assets/ directory but no other directories are synchronised between nodes.
```

Comment:
Is this always true on CWP, or is it possible for SilverStripe 4 to use the `public` folder?
The latter would look like `./www/public/assets`
But I'm not sure of the server configuration to say that would/could ever be the case.

Reply:
I didn't think very hard about the actual on disk location of assets I'll need to do some research to understand how that works and I'll get back to you with an updated commit

Reply:
I had a quick look at our setup and I'll need another PR for the 2.X line to include public webroot
My understanding is following for SS3 & SS4 on cwp
SS3 cwp1.X /sites/<Stackid>/www/assets
SS4 cwp2.X /sites/<Stackid>/www/public/assets

Reply:
Created specific PR for 2.0 onwards
https://github.com/silverstripe/cwp/pull/165

Reply:
As it turns out we just use different sim-links to achieve this

Reply:
Ideal, thanks for following up :)